openedx/core/djangoapps/models/course_details.py

```diff
@@ -285,6 +290,10 @@ def update_from_json(cls, course_key, jsondict, user):  # pylint: disable=too-ma
 
         cls.update_about_video(descriptor, jsondict['intro_video'], user.id)
 
+        # fires the signal that course pacing has changed after changes are reflected in db
```

Comment:
move this to L282 –  `# Fire a signal indicating that the course pacing has changed.`